deluge/argparserbase.py

```diff
@@ -326,7 +326,7 @@ def _handle_ui_options(self, options):
             # Write pid file before chuid
             if options.pidfile:
                 with open(options.pidfile, 'wb') as _file:
-                    _file.write('%d\n' % os.getpid())
+                    _file.write(b'%d\n' % os.getpid())
```

Comment:
Actually the pidfile should be opened in text mode so revert this and change `wb` to `w` above